src/comodo/jaxsimSimulator/jaxsimSimulator.py

```diff
@@ -0,0 +1,465 @@
+from __future__ import annotations
+
+import logging
+import math
+import pathlib
+from enum import Enum
+from typing import Any, Optional, Tuple
```

Comment:
Optional is deprecated from [PEP 604](https://peps.python.org/pep-0604/) and `Tuple` is deprecated from [PEP 585](https://peps.python.org/pep-0585/)
```suggestion
from typing import Any
```